frontend/src/types/post-type.tsx

```diff
@@ -6,6 +6,7 @@ type Post = {
   timeOfPost: string;
   description: string;
   categories: string[];
+  authorId?: string;
 };
```

Comment:
authorId? -> authorId
Okay now I know why you used `?` 

For this if we require to change and refill the db we will 
Let's not have `?`

Reply:
some posts in db does not have authorId that's why it's being added optional

Reply:
@devsharmagit so said
> For this if we require to change and refill the db we will

Let's add to everything and change the sample-posts.json wala file